pkg/rebuild/maven/infer.go

```diff
@@ -283,7 +283,7 @@ func findPomXML(commit *object.Commit, pkg string) (*PomXML, string, error) {
 	var names []string
 	var pomXMLs []PomXML
 	commitTree.Files().ForEach(func(f *object.File) error {
-		if !strings.HasSuffix(f.Name, "pom.xml") {
+		if filepath.Base(f.Name) != "pom.xml" || strings.Contains(f.Name, "test/") {
```

Comment:
I think we may want to use path instead of filepath (I think git metadata will always use forward slash?)

Reply:
> I think git metadata will always use forward slash?

I was not sure about this.[ `File` struct](https://github.com/go-git/go-git/blob/39261590f7f396873e83e7fc8ca8453f71225f10/plumbing/object/file.go#L15) did not specify this.

Reply:
Yeah off the top of my head I'm not sure. You could dig into the implementation to see how it's set since the git object itself will only have the last level file path (the remainder of the path elements are constructed from the parent tree objects).

Reply:
It's an option. Maybe open an issue where we could discuss?